packages/@coorpacademy-components/src/molecule/skill-picker-modal/index.js

```diff
@@ -0,0 +1,142 @@
+import React, {useMemo, useState, useEffect, useCallback} from 'react';
+import PropTypes from 'prop-types';
+import BaseModal from '../base-modal';
+import Chip from '../../atom/chip';
+import Loader from '../../atom/loader';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const SkillPickerModal = (props, context) => {
+  const {
+    skills,
+    isOpen,
+    isLoading,
+    maxSelectedSkills = 6,
+    minSelectedSkills = 3,
+    onCancel,
+    onConfirm,
+    onClose
+  } = props;
+  const {translate} = context;
+
+  const [skillList, setSkillList] = useState([]);
+
+  const handleCancel = useCallback(() => {
+    setSkillList(JSON.parse(JSON.stringify(skills)));
+    onCancel();
+  }, [setSkillList, skills, onCancel]);
+
+  const handleClose = useCallback(() => {
+    setSkillList(JSON.parse(JSON.stringify(skills)));
+    onClose();
+  }, [setSkillList, skills, onClose]);
+
+  const selectedSkills = useMemo(() => skillList.filter(skill => skill.focus), [skillList]);
+  const isError = useMemo(
+    () => selectedSkills.length > maxSelectedSkills,
+    [selectedSkills, maxSelectedSkills]
+  );
+
+  const footer = useMemo(() => {
+    const footerDescription = isError
+      ? translate('skill_focus_footer_error_description').replace(
+          '.',
+          selectedSkills.length - maxSelectedSkills
+        )
+      : translate('skill_focus_footer_description').replace(
```

Comment:
Don't we have interpolation in place for locales?

Reply:
Normally yes,
in global.json, you can use :
```
"Select at least {{your_var}} more skills"
```
and here:
```
translate('skill_focus_footer_description', {your_var: value})
```